lib/widgets/tb_elevated_button.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:tech_byte/utils/colors.dart';
+import 'package:tech_byte/utils/constants.dart';
+
+enum TBButtonType { primary, secondary }
+
+class TBButton extends StatelessWidget {
+  final void Function() onPressed;
+  final String text;
+  final bool enabled;
+  final TBButtonType type;
+  final bool isLoading;
+
+  const TBButton({
+    super.key,
+    required this.onPressed,
+    required this.text,
+    required this.type,
+    this.enabled = true,
+    this.isLoading = false,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: kButtonHeight,
+      width: double.infinity,
```

Comment:
Never use fixed dimensions unless it is necessary. Reach the goal with paddings or margins which are always fixed. With this approach, there is a problem when the user changes the default font of the mobile.
<img width="426" alt="image" src="https://github.com/damjan-Igrowth/INT-Milan_Zezelj-Shop/assets/119666343/abdca1be-4c87-4079-8638-499c9e6795ad">